src/yocto/getting-started.md

```diff
@@ -0,0 +1,26 @@
+# Getting started
+More information can be found on the official project website: https://www.yoctoproject.org/.
+## What is?
+First of all, you have to understand that this is not a Linux distribution. It is an open-source project that provides developers with useful tools such as software stacks, layers, templates, and methods, to create a custom Linux-based system from scratch for different hardware architectures, this allows developers to boot on different embedded hardware, making embedded Linux simpler and more accessible.  
+
+## Why is this important?
```

Comment:
An example would help to give more context about why and when to implement yocto. maybe showcase a project or link to some demo or project using yocto and highlight characteristics.